app/models/setting/katello.rb

```diff
@@ -20,7 +20,7 @@ def self.load_defaults
         self.set('pulp_sync_node_action_finish_timeout', N_("Time in seconds to wait for a pulp node to finish sync"), 12.hours.to_i),
         self.set('check_services_before_actions', N_("Whether or not to check the status of backend services such as pulp and candlepin prior to performing some actions."), true),
         self.set('force_post_sync_actions', N_("Force post sync actions such as indexing and email even if no content was available."), false),
-        self.set('default_download_policy', N_("Default download policy for repositories (either 'immediate', 'on_demand', or 'background')"), "on_demand"),
+        self.set('default_download_policy', N_("Default download policy for repositories"), "on_demand"),
```

Comment:
I'm wondering if we should remove the options from the description. Would be useful to see the options in the CLI which displays the setting description.